ci_scripts/utils/filesystem_helpers.py

```diff
@@ -0,0 +1,22 @@
+"""Helpers with regards to actions on the filesystem."""
+import os
+from contextlib import contextmanager
+
+
+@contextmanager
+def cd(new_dir):
+    """Changes directory.

```

Comment:
I'm not entirely sure on the doc sytax, so I'm going to propose change in comment instead of suggestion.

```
"""Context manager allowing an operation to be performed inside given directory.
```

Stating that it's context manager is explicit enough and you won't need to say `use as...`.